pyproject.toml

```diff
@@ -65,3 +65,11 @@ extension="html"
 blank_line_before_tag="block,if"
```

Comment:
regarding #236, can we not just remove the if here, so that no blank line is added before if statements?

Reply:
Sure, but it took us months to get to this basic set of `djlint` settings